cgi-bin/LJ/Console/Command/Suspend.pm

```diff
@@ -45,6 +45,11 @@ sub execute {
     my $remote = LJ::get_remote();
     my $openid_only =
         $remote->has_priv( "suspend", "openid" ) && !$remote->has_priv( "suspend", "*" );
+    my $recent_only =
+        $remote->has_priv( "suspend", "recent" ) && !$remote->has_priv( "suspend", "*" );
+
+    return $self->error("You can only suspend users using /admin/recent_accounts.")
+        if $recent_only;
 
```

Comment:
I think this needs to handle the case where someone can suspend both recent and open ID, but does not have Global suspend.